### PR TITLE
docs(vital-list): Add Vital List documentation

### LINKED ISSUE #1729: [DOC] Vital List documentation
## User Story
AS AN end user of the Bodiless documentation,
I WANT to understand how to use the Vital List package,
SO THAT I can add lists to my site.

## Acceptance Criteria
1. The Documentation is updated to include how a Content Editor can use the Vital List component in the editor interface.
2. The Documentation is updated to include how a Site Builder can configure the Vital List component.

### PR DESCRIPTION
**Pull request by misterjones**
## Changes
- Add documentation for Vital List component.
  - Update package README to be consistent with others.
  - Add screenshots.
- Minor fixes to "Vital Component Template" page.
- Add Vital List package to Test Site so that Vital List documentation
  is generated for it.

## Related Issues

Resolves #1729

### CODE REVIEW

File: packages/vital-list/doc/README.md [hunk 1]
```diff
@@ -1,15 +1,111 @@
...
-import Helmet from 'react-helmet';
-import { as } from '@canvasx/elements';
-import { vitalMeta } from '@canvasx/meta';
+Vital List provides four categories of Lists:
 
-const SeoMetaHelmet = as(vitalMeta.SEO)(Helmet);
-const ShareMetaHelmet = as(vitalMeta.Share)(Helmet);
-```
\ No newline at end of file
+- [Inforgraphic](#infographic)
```
Comment (hvanyo) [inline packages/vital-list/doc/README.md:12]:
```suggestion
- [Infographic](#infographic)
```

File: packages/vital-list/doc/README.md [hunk 2]
```diff
@@ -1,15 +1,111 @@
...
+        like other Image components.
+      - You can select your text and format it via the [Rich Text
+        Editor](/Components/Editors/RichText) toolbar.
+    - **[Links](#links):**
+      - You can select the text and [add a link](/Components/Link/#add-a-link-to-a-component).
+    - **[Rich Text](#rich-text):**
+      - You can select your text and format it via the [Rich Text
+        Editor](/Components/Editors/RichText) toolbar.
+
+## Site Builder Details
```
Comment (hvanyo) [inline packages/vital-list/doc/README.md:109]: Add the shadowing blurb here.

File: packages/vital-list/doc/README.md [hunk 3]
```diff
@@ -1,15 +1,111 @@
...
+    - Hover over the "i" icon for information about a particular list variation.  
+    ![Add a Vital List Component](./assets/AddVitalListComponent.jpg)
+01. [Edit your list](#edit-a-list) as desired.
+
+### Edit a List
+
+After [adding a Vital List](#add-a-list), you can add list items to it:
+
+01. While in [Edit Mode](/ContentEditorUserGuide/#edit-mode), select the placeholder (or existing)
+    text of the desired list item, and being typing to make your edits.
```
Comment (hvanyo) [inline packages/vital-list/doc/README.md:94]:
```suggestion
    text of the desired list item, and begin typing to make your edits.
```

File: packages/vital-list/doc/README.md [hunk 4]
```diff
@@ -1,15 +1,111 @@
...
+- [Rich Text](#rich-text)
+- [Plain Text](#plain-text)
+
+Excluding Infographic Lists, each category offers the following variations:
+
+- **Bullets:** A bulleted list, as you'd associate with a typical unordered list element (`<ul>`).
+- **Numbers:** A numbered list, as you'd associate with a typical ordered list element (`<ol>`).
+- **Undecorated:** A simple list without any item markers.
+
+?> **Note:** Out of the box, Vital Lists are _single-level-only_ — i.e., they do not support nested
```
[the remark this answers is not in the document]
Comment (misterjones) [reply packages/vital-list/doc/README.md]: ✅  Noted!

File: packages/vital-list/doc/README.md (hunk 2 above)
Comment (misterjones) [reply packages/vital-list/doc/README.md:109]: ✅  Added.

File: packages/vital-list/doc/README.md [hunk 5]
```diff
@@ -1,15 +1,111 @@
...
+
+To add a Vital List Component to your page:
+
+01. While in [Edit Mode](/ContentEditorUserGuide/#edit-mode), select a [Flow
+    Container](/Components/FlowContainer/), and, within its context menu, under "Component," click
+    **Add** (or **Swap**, if you're replacing a component).
+01. In the "Insert Component" modal, under "Type," check the "List" checkbox, and select the type
+    of list you would like to add.
+    - Vital List provides the following list options by default:
+      - [Inforgraphic](#infographic)
```
[the remark this answers is not in the document]
Comment (misterjones) [reply packages/vital-list/doc/README.md]: Haha, really hard to fight that muscle-memory of typing "information," apparently. 😆

**Review by hvanyo — DISMISSED** [review]

File: packages/vital-list/doc/README.md [hunk 6]
```diff
@@ -1,15 +1,118 @@
...
+## Site Builder Details
+
+At the site or global regional/brand library level, you can use the Vital List component as is, or
+extend/shadow the existing component.
+
+### Customizing Via Shadowing (*Preferred Method)
+
+Define a Shadowing token collection as defined in [Shadow](./VitalElements/Shadow).
+
+File to shadow: `./lib/shadow/vital-list/{MyList}.js`
```
Comment (dewen) [inline packages/vital-list/doc/README.md:118]: Wonder what's the relative path to project root, I thought shadow files are under
`/packages/__vital__/lib/shadow/@bodiless/`

Comment (hvanyo) [reply packages/vital-list/doc/README.md:118]: @misterjones  -> is hte PR where I was doing testing of shadowing:  https://github.com/lucas-varela/Bodiless-JS/pull/29
i.e.
packages/__vital__/src/shadow/@bodiless/vital-card/Card.ts
__vital__ is replaced with your package name..

**Review by hvanyo — APPROVED** [review]

**Review by dewen — APPROVED** [review]

**Pull request merged**